### PR TITLE
feat: Update menu

### PR DESCRIPTION
**Pull request by hachiojidev**

### CODE REVIEW

File: src/components/Menu/config/config.ts [hunk 1]
```diff
@@ -52,14 +52,22 @@ const config: (t: ContextApi['t']) => ConfigMenuItemsType[] = (t) => [
   },
```
Comment (RabbitDoge) [inline src/components/Menu/config/config.ts:52]: /config/config.ts ?

File: src/views/Nft/market/Collectibles/index.tsx [hunk 2]
```diff
@@ -31,8 +31,9 @@ const Collectible = () => {
   return (
     <>
       <Page>
-        <Link to="/nft/market/collectibles/pancake-bunnies">Pancake Bunnies</Link>
-        <Link to="/nft/market/collectibles/pancake-squad">Pancake Bunnies</Link>
+        <Link to="/nft/market/collections/pancake-bunnies">Pancake Bunnies</Link>
+        <br />
+        <Link to="/nft/market/collections/pancake-squad">Pancake Squad</Link>
```
Comment (RabbitDoge) [inline src/views/Nft/market/Collectibles/index.tsx:36]: The squad won't be available at launch

Comment (hachiojidev) [reply src/views/Nft/market/Collectibles/index.tsx:36]: Noted

File: src/components/Menu/config/config.ts (hunk 1 above)
Comment (hachiojidev) [reply src/components/Menu/config/config.ts:52]: This wasn't used anywhere so I removed it. It might have been leftover from the rebase.

Comment (hachiojidev) [reply src/components/Menu/config/config.ts:52]: Yeah maybe we should change it. It is what is on develop right now.

Comment (0xCorgi) [reply src/components/Menu/config/config.ts:52]: In this folder there are 2 config file, one for the menu and one for the footer.
Maybe we should rename this one to menuConfig then ?

**Review by RabbitDoge — APPROVED** [review]

**Review by ChefHutch — APPROVED** [review]

**Pull request merged**